userguide/virtualmachines.rst

```diff
@@ -657,116 +660,112 @@ contain a space.
 
 .. figure:: images/virtual-machines-docker-devices-rawfile.png
 
-   Edit Rancher Password in Raw File Device
+   Changing the Docker Host Password
 
 
 Start the Docker Host
 ~~~~~~~~~~~~~~~~~~~~~
 
 Go to
-:menuselection:`Virtual Machines`,
-then click on the red |ui-power| button of the Docker Host to start it.
-
-Starting a Docker Host can take some time. Connecting to the Serial
-Shell is possible during the startup process to view the activity of the
-Docker Host. When a message about :literal:`RancherOS` starting appears
-and the shell stops posting new messages, press :kbd:`Enter` to see the
-:literal:`ClientHost login:` text and continue to log in.
-
-
-SSH into the Docker Host
-~~~~~~~~~~~~~~~~~~~~~~~~
-
-It is possible to SSH into a running Docker Host. Go
-to the
-:menuselection:`Virtual Machines` page and find the card for the Docker
-Host. The card shows the :guilabel:`Com Port` for the Docker Host. In
-this example, :literal:`/dev/nmdm12B` is used.
-
-Use an SSH client to connect to the %brand% server. Remember this also
-requires the :ref:`SSH` service to be running. Depending on the %brand%
-system configuration, it might also require changes to the
-:guilabel:`SSH` service settings, like setting
-:guilabel:`Login as Root with Password`.
-
-At the %brand% console prompt, connect to the Rancher Host with
-`cu <https://www.freebsd.org/cgi/man.cgi?query=cu>`__, replacing
-:samp:`{/dev/nmdm12B}` with the value from the Docker Host
-:guilabel:`Com Port`:
-
-.. code-block:: none
-
-   cu -l /dev/nmdm12B -s 9600
-
+:menuselection:`Virtual Machines`
+and click
+|ui-options| :menuselection:`--> Start`
+to boot the Docker Host.
 
-If the terminal does not show a :literal:`rancher login:` prompt,
-press :kbd:`Enter`. The Docker Host can take some time to start and
-display the login prompt.
+A Docker Host can take some time to boot. Connect to the Serial Shell
+during the startup process to view the activity of the Docker Host.
 
+The :literal:`RancherOS` graphic is shown. Press :kbd:`Enter` to see the
```

Comment:
RancherOS should not be a literal here, it is not something the user will type.
"graphic" is probably console graphics, should probably be more clear about that, or just skip mentioning it at all.